<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagementPage</class>
 <widget class="QWidget" name="ManagementPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>891</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="LargeTitleLabel" name="LargeTitleLabel">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>827</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="HorizontalSeparator" name="HorizontalSeparator"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="SimpleCardWidget" name="SimpleCardWidget">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="CardWidget" name="CardWidget">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="LineEdit" name="idTbx">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>学号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LineEdit" name="nameTbx">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>姓名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LineEdit" name="majorTbx">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>专业</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LineEdit" name="gradeTbx">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>班级</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LineEdit" name="ageTbx">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>年龄</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LineEdit" name="gpaTbx">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>成绩</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="PrimaryToolButton" name="clearSearchBtn"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="TableWidget" name="TableWidget">
        <property name="rowCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>140</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>45</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>学号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>姓名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>专业</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>班级</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>年龄</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>成绩</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="PrimaryPushButton" name="queryBtn">
          <property name="text">
           <string>查找</string>
          </property>
          <property name="hasIcon" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PrimaryPushButton" name="addBtn">
          <property name="text">
           <string>添加</string>
          </property>
          <property name="hasIcon" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PrimaryPushButton" name="deleteBtn">
          <property name="text">
           <string>删除</string>
          </property>
          <property name="hasIcon" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PrimaryPushButton" name="exportBtn">
          <property name="text">
           <string>导出</string>
          </property>
          <property name="hasIcon" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="TransparentToolButton" name="leftArrowBtn"/>
        </item>
        <item>
         <widget class="BodyLabel" name="pageIndexLabel">
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TransparentToolButton" name="rightArrowBtn"/>
        </item>
        <item>
         <widget class="BodyLabel" name="pageCountLabel_2">
          <property name="text">
           <string>共</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BodyLabel" name="pageCountLabel">
          <property name="text">
           <string> %d </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BodyLabel" name="pageCountLabel_3">
          <property name="text">
           <string>页</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="BodyLabel" name="BodyLabel_3">
          <property name="text">
           <string>跳转</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEdit" name="goToPageTbx">
          <property name="placeholderText">
           <string>页码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BodyLabel" name="BodyLabel_4">
          <property name="text">
           <string>页</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TransparentToolButton" name="gotoBtn"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryToolButton</class>
   <extends>ToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TransparentToolButton</class>
   <extends>ToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LargeTitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
